Monad/extractFunctor

```diff
@@ -0,0 +1,11 @@
+    let Functor = ./../Functor/Type
+
+in  let Monad = ./Type
+
+in  let extractFunctor = ./../Applicative/extractFunctor
+
+in  let extractApplicative = ./../Monad/extractApplicative
+
+in    λ(f : Type → Type)
+    → λ(monad : Monad f)
+    → extractFunctor f (extractApplicative f monad) : Functor f
```

Comment:
Does it make sense to write `Monad`'s `extractFunctor` in terms of `Applicative`'s `extractFunctor`?

It's less circuitous to just construct the dictionary here a la `monad.{ map }`

Reply:
Yeah, I think `.{map}` is fine.